client/src/components/reservation.jsx

```diff
@@ -1,8 +1,30 @@
 import React from 'react';
 import ReservationForm from './reservationForm.jsx';
 import styles from './reservation.module.css';
+import {ajax} from 'jquery';
```

Comment:
highly recommend using axios or fetch instead of jquery.  Jquery is a heavy library with a lot of redundant functionality that you'll never be using.  Axios is a much smaller dependency